ts/api/provider_frame_elem.ts

```diff
@@ -29,19 +31,38 @@ const DEFAULT_FRAME_CSS = `
   position: fixed;
   border: none;
   z-index: 9999;
+}
+
+.openyolo-visible.bottomSheet {
   bottom: 0;
+  width: 100%;
+  height: 320px;
+}
+
+.openyolo-visible.navPopout {
+  top: 0;
+  width: 100%;
+  height: 320px;
+}
+
+.openyolo-visible.fullScreen {
+  top: 0;
   left: 0;
   width: 100%;
+  height: 100%;
 }
 
-@media (min-width:801px) {
-  .openyolo-visible {
-    position: fixed;
-    left: auto;
+@media only screen and (min-device-width:321px) {
+  .openyolo-visible.bottomSheet {
```

Comment:
Fair enough, I can simplify the CSS to assume that bottomSheet is for mobile only (always 100% width), navPopout is for desktop only (always tied to top-right). We can reintroduce responsiveness if we can give a clear enough definition of the expected behavior.